### DISCUSSION TITLE
Accidental swipes

### DISCUSSION THREAD

**Discussion by JP-Reitsma** [Nachtzuster/BirdNET-Pi#193] [General]
First of all: I love this version of birdNET, thanks for all the effort and updates! I enjoy comparing the charts as it's very interesting to study.

It's very hard to zoom in and pan around on the daily charts with my phone, without accidentally swipe to the next (or previous) day.
If there could be a button to turn off this swipe feature, would be awesome. 
Just two buttons at top and/or at bottom of the chart to move to the next/previous day would be a perfect alternative imo.

**Comment by alexbelgium** [comment]
Hi, here is the code that implements the swipe : either we could add an option in the advanced settings or you simply comment it out in the relevant pages https://github.com/mcguirepr89/BirdNET-Pi/pull/1034/files

**Reply by JP-Reitsma** [reply]
Thanks for that helpful information, I will comment it out for now. 
I guess time will tell if I'm the only one struggling with the accidental swipe...
I know a little bit of coding so I can make the changes on the rpi, no problem. But not super familiar with github and the update process so I hope you can give me some advice. Do I have to make the changes every time I install the birdNET updates or can I avoid that those changes come undone?

**Reply by alexbelgium** [reply]
I think it's for every updates, which mean that you could write a sed script to comment it out automatically at device boot for example...

Not sure actually @lloydbailley do you know if manual modifications get overwritten upon updates?

**Comment by Nachtzuster** [comment]
@JP-Reitsma I recently pushed a [fix](https://github.com/Nachtzuster/BirdNET-Pi/commit/98f0d04329881b11fd147594d4a040bc1de7f5a7) that should improve swipe vs scroll behaviour - give it a try?

**Reply by JP-Reitsma** [reply]
Thanks, it does help a little bit to reduce the occurrence. However, the issue remains on my phone, specifically when zoomed in and panning between the morning hours and the evening. My workaround is avoiding the panning and zooming out and back in on the next section which is really inconvenient.
Thinking about this made me see the similarity with the gallery apps on your phone where you can't swipe to a next photo without zooming back out first. If it would work like that, it would be an elegant intuitive solution.
